### ISSUE TITLE
can not access svg or svg child elements

### ISSUE THREAD

**Issue by matti** [teamcapybara/capybara#2048]
latest versions (chromedriver 2.40, capybara 3.2.1)

```ruby
visit "https://jsfiddle.net/gh/get/jquery/1.7.2/highslide-software/highcharts.com/tree/master/samples/highcharts/boost/line/"

sleep 5

within_frame "result" do
  sleep 5
  svg = find "div svg"
  p svg
end
```

explodes

```
capybara-3.2.1/lib/capybara/selenium/node.rb:175:in `path': undefined method `+' for nil:NilClass (NoMethodError)
```

and the problem is in `siblings.index(node)` because `siblings` is `[]`
```ruby
        selector += "[#{siblings.index(node) + 1}]" unless siblings.size == 1
```

I managed to monkeypatch svg finding to work, but children ("div svg *") will still explode:

```ruby
  def path
    path = find_xpath(XPath.ancestor_or_self).reverse

    result = []
    while (node = path.shift)
      parent = path.first
      selector = node.tag_name
      if parent
        siblings = parent.find_xpath(selector)

        # MONKEY
        if selector == "svg" && siblings.size == 0
          siblings = parent.find_xpath "//*[local-name() = 'svg']"
        end
        # MONKEY

        selector += "[#{siblings.index(node) + 1}]" unless siblings.size == 1
      end
      result.push selector
    end

    '/' + result.reverse.join('/')
  end
```

**Comment by twalpole** [comment]
Technically this doesn't prevent you from accessing the element or it's children -- it does however prevent you from outputting the description of the element. 

    p svg[:class] => highcharts-root
    p svg.all('*').size => 55

**Comment by twalpole** [comment]
@matti Please try the `namespace_path` branch and see if that works for you

**Comment by twalpole** [comment]
Closed via PR #2049
